INSTALLING.md

```diff
@@ -2,199 +2,178 @@
-> that all the packages in their environment are compatible with each other.
->
-> If you do choose to manage your own environment, please note that you must
-> use Pip 20.1.1 or lower to install this project. This is the highest version
-> of Pip that supports editable installs in the way that we use it.
+---
 
-#### Option 2: From wheel
+## TT-Metalium Installation:
 
-Download the latest wheel from our
-[releases](https://github.com/tenstorrent/tt-metal/releases/latest) page for
-the particular Tenstorrent card architecture that you have installed on your
-system. (ie. Grayskull, Wormhole, etc)
+> [!IMPORTANT]
+>
+> If you are using a release version of this software, check installation insttructions packaged with it.
 
-Install the wheel using your Python environment manager of choice. For example,
-to install with `pip`:
+### Step 1. Clone the Repository:
 
 ```sh
-pip install <wheel_file.whl>
+git clone https://github.com/tenstorrent/tt-metal.git --recurse-submodules
+cd tt-metal
+git submodule foreach 'git lfs fetch --all && git lfs pull'
 ```
 
-1. (For models users only) Set up environment for models
-
-If you are going to try our pre-built models in `models/`, then you must execute
-the following to:
-
-- install their required dependencies
-- set appropriate environment variables
-- set the CPU performance governor to ensure high performance on the host
-
+### Step 2. Setup environment variables and invoke our build scripts:
+
+- Note: Some advanced build configurations like unity builds require `CMake 3.20`.
+  To install `CMake 3.20` run:
+    ```sh
+    sudo apt remove cmake -y
+    pip3 install cmake --upgrade
+    hash -r
+    cmake --version
+    ```
+- For Grayskull:
 ```sh
+export ARCH_NAME=grayskull
+export TT_METAL_HOME=$(pwd)
 export PYTHONPATH=$(pwd)
-pip install -r tt_metal/python_env/requirements-dev.txt
-sudo apt-get install cpufrequtils
-sudo cpupower frequency-set -g performance
+./build_metal.sh
 ```
 
-#### Option 3: From Docker Release Image
-
-Download the latest Docker release from our [Docker registry](https://github.com/orgs/tenstorrent/packages?q=tt-metalium-ubuntu&tab=packages&q=tt-metalium-ubuntu-20.04-amd64-release) page for
-the particular Tenstorrent card architecture that you have installed on your
-system. (ie. Grayskull, Wormhole, etc)
-
+- For Wormhole:
 ```sh
-docker pull ghcr.io/tenstorrent/tt-metal/tt-metalium-ubuntu-20.04-amd64-release/<arch_name>:latest-rc
-docker run --it --rm -v /dev/hugepages-1G:/dev/hugepages-1G --device /dev/tenstorrent ghcr.io/tenstorrent/tt-metal/tt-metalium-ubuntu-20.04-amd64-release/<arch_name>:latest-rc bash
+export ARCH_NAME=wormhole_b0
+export TT_METAL_HOME=$(pwd)
+export PYTHONPATH=$(pwd)
+./build_metal.sh
 ```
-where `arch_name` is one of `grayskull`, `wormhole_b0`, or `blackhole`,
-depending on your Tenstorrent card type.
 
-When inside of the container,
+- For Blackhole:
 ```sh
-python3 -c "import ttnn"
```

Comment:
Could we use the same verification step here instead of `python3 -c "import ttnn"` use `python3 -m ttnn.examples.usage.run_op_on_device`

Reply:
`python3 -m ttnn.examples.usage.run_op_on_device` is suggested right after that. Is `python3 -c "import ttnn"` not a required step for the docker install?

Reply:
No it was a validation step as well, but I'd rather have the same validation step as for the wheel.